frontend/src/metabase/admin/permissions/components/ToolbarUpsell/ToolbarUpsell.unit.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+import ToolbarUpsell from "./ToolbarUpsell";
+
+const setup = () => {
+  renderWithProviders(<ToolbarUpsell />);
+};
+
+describe("ToolbarUpsell", () => {
+  it("should add utm_media to the upgrade link", () => {
+    setup();
+
+    userEvent.click(screen.getByText("Get more control"));
+
+    expect(
+      screen.getByRole("link", { name: "Upgrade to Pro or Enterprise" }),
+    ).toHaveAttribute("href", expect.stringContaining("permissions_top"));
```

Comment:
Same as other comment whether we should test `utm_media=permissions_top` instead?